Assets/Scripts/Dungeon/Room.cs

```diff
@@ -39,6 +39,9 @@ public class Room : MonoBehaviour
         [SerializeField]
         private Spawners.Spawner[] entitiesToSpawn;
 
+        /// <summary> Array of enemy types that can be spawned in generated room </summary>
+        public Enums.EnemyTypes[] proceduralEnemies;
```

Comment:
This behavior shouldn't be here.  If you were trying to get a list of the active enemies then it should be added as a method to the EnemyPool class where that info is actually available.  You also shouldn't be worrying about this kind of thing.  Its nice that you are thinking ahead, but this is better handled with managers and global data I will be writing in the coming few weeks.